web/packages/teleterm/electron-builder-config.js

```diff
@@ -178,12 +183,20 @@ module.exports = {
         from: env.CONNECT_TSH_BIN_PATH,
         to: './bin/tsh.exe',
       },
+      env.CONNECT_WINTUN_DLL_PATH && {
+        from: env.CONNECT_WINTUN_DLL_PATH,
+        to: './bin/wintun.dll',
+      },
     ].filter(Boolean),
   },
   nsis: {
     // Turn off blockmaps since we don't support automatic updates.
     // https://github.com/electron-userland/electron-builder/issues/2900#issuecomment-730571696
     differentialPackage: false,
+    // Use a per-machine installation to support VNet.
+    // VNet installs a Windows service per-machine, and tsh.exe must be
+    // installed in a path that is not user-writable.
+    perMachine: true,
```

Comment:
The path to the executable on Windows in the docs needs to be updated.

* https://goteleport.com/docs/connect-your-client/teleport-connect/#submitting-an-issue
* https://goteleport.com/docs/connect-your-client/teleport-connect/#skipping-tls-certificate-verification
* https://goteleport.com/docs/connect-your-client/teleport-connect/#uninstalling-teleport-connect

On branch/v17, we should keep both paths, for pre 17.3 versions and >= 17.3.

Reply:
The installation section could mention the permissions needed too. Running a silent install from an unprivileged cmd.exe automatically prompts for privileges. Running a silent install from a privileged cmd.exe works as expected.

* https://goteleport.com/docs/connect-your-client/teleport-connect/#installation--upgrade

Reply:
I will do another PR with docs updates thanks for pointing these out